# HG changeset patch
# User Rik <rik@octave.org>
# Date 1511720559 28800
#      Sun Nov 26 10:22:39 2017 -0800
# Node ID 4dce9d03b2ba1896b14f2912fa8835e02479b5c4
# Parent  1262d7c4712ed3fc9d3a7c38a610b5eb4557feec
Remove regression mark for second BIST test for textscan bug #52479.

* file-io.cc: Remove regression mark for textscan BIST test where delimiter is
a whitespace, but presented as a cellstr array rather than a string.

diff --git a/libinterp/corefcn/file-io.cc b/libinterp/corefcn/file-io.cc
--- a/libinterp/corefcn/file-io.cc
+++ b/libinterp/corefcn/file-io.cc
@@ -2211,17 +2211,17 @@ as the name of the function when reporti
 ## Check for delimiter after exponent
 %!assert (textscan ("1e-3|42", "%f", "delimiter", "|"), {[1e-3; 42]})
 
 %!test <*52479>
 %! str = "\t\ta\tb\tc\n";
 %! ret = textscan (str, "%s", "delimiter", "\t");
 %! assert (ret, { {''; ''; 'a'; 'b'; 'c'} }) ;
 
-%!test <*52479>
+%!test <52479>
 %! str = "\t\ta\tb\tc\n";
 %! ret = textscan (str, "%s", "delimiter", {"\t"});
 %! assert (ret, { {''; ''; 'a'; 'b'; 'c'} }) ;
 
 */
 
 // These tests have end-comment sequences, so can't just be in a comment
 #if 0
